### PR TITLE
Add Dockerfile and update readme

### PR DESCRIPTION
**Pull request by Rithwik0480**

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -0,0 +1,12 @@
+FROM node:18-slim
+
+LABEL maintainer="engineering@dhiway.com"
+
+WORKDIR /demo
+COPY . /demo
+
+RUN npm install
+
+EXPOSE 30333 9933 9944 9615
```
Comment (amarts) [inline Dockerfile]: We don't need this. Demo script is a client side process. No PORTS need to be open.

File: Dockerfile [hunk 2]
```diff
@@ -0,0 +1,12 @@
...
+
+ENTRYPOINT [ "npm" , "run" , "demo" ]
```
Comment (amarts) [inline Dockerfile]: can we just provide like below and ask the users to provide third option in command line? 
```suggestion
ENTRYPOINT [ "npm" , "run" ]
```

**Comment by amarts** [comment]
also its a good idea to make the branching in your repo to suit the task, and not push changes to `develop` branch.

**Comment by Rithwik0480** [comment]
closing this PR as I have raised a new one from another branch 👉 https://github.com/dhiway/cord-demo-scripts/pull/36

File: Dockerfile (hunk 2 above)
Comment (Rithwik0480) [reply Dockerfile]: docker run command is updated to `docker run <image-name> demo`

**Pull request closed without merging**